sufia-models/app/services/sufia/generic_file_indexing_service.rb

```diff
@@ -6,6 +6,7 @@ def generate_solr_document
         solr_doc[Solrizer.solr_name('file_format')] = object.file_format
         solr_doc[Solrizer.solr_name('file_format', :facetable)] = object.file_format
         solr_doc['all_text_timv'] = object.full_text.content
+        solr_doc[Solrizer.solr_name('file_size', Solrizer::Descriptor.new(:integer, :stored);)] = object.content.size.to_i
```

Comment:
The semi colon has been eradicated. Regarding the constant is your concern over performance or just aesthetics? Until STORED_INTEGER_DESCRIPTOR is used in other places the performance difference is neglible as can be seen if you run the following script in a Pry console

@constant = Solrizer::Descriptor.new(:integer, :stored)
def createNewDescriptorWithConstant
	Solrizer.solr_name('file_size', @constant)
end

def createNewDescriptorDynamically
	Solrizer.solr_name('file_size', Solrizer::Descriptor.new(:integer, :stored))
end

Benchmark.bmbm do |x|
	x.report("Constant : ") { 500.times { createNewDescriptorWithConstant }}
	x.report("Dynamic : ") { 500.times { createNewDescriptorDynamically }}
end